pkg/kubernetes/storageclass/v1alpha1/storageclass.go

```diff
@@ -0,0 +1,110 @@
+package v1alpha1
+
+import (
+	storagev1 "k8s.io/api/storage/v1"
+)
+
+// StorageClass is a wrapper over API based
+// storage class instance
+type StorageClass struct {
+	object *storagev1.StorageClass
```

Comment:
You should export `object` as this will be use outside of this package.

Reply:
Pushing a new commit will do in that commit